ComSemApp/templates/ComSemApp/student/assessment.html

```diff
@@ -2,5 +2,22 @@
 
 {% block content %}
 <h1>Speaking Practice View</h1><br>
+
+<script>
+    var given_time = 15;
+    var current_time =15;
+    var question_timer = setInterval(function(){
+      if(current_time <= 0){
+        clearInterval(question_timer);
+      }
+      document.getElementById("timer_bar").value = given_time - current_time;
+      document.getElementById("text_timer").textContent = current_time;
+      current_time -= 1;
+    }, 1000);
+</script>
```

Comment:
Is there a way that we can make the timer bar decrease continuously, rather than ticking each second?